libc/test/src/unistd/pipe2_test.cpp

```diff
@@ -46,7 +46,7 @@ TEST(LlvmLibcPipe2Test, ReadAndWriteViaPipe2) {
   ASSERT_ERRNO_SUCCESS();
 }
 
-TEST(LlvmLibcPipe2Test, Pipe2InvalidFlags) {
+TEST(LlvmLibcPipe2Test,Pipe2InvalidFlags) {
```

Comment:
The linter is not happy with this change. You can do `git clang-format HEAD~<N>` to format `<N>` commits. `<N>` is optional when `N == 1`.  For example, `git clang-format HEAD~5` would format that past 5 commits.